### ISSUE TITLE
Type Provider issues on OSX

### ISSUE THREAD

**Issue by dvitonis** [ionide/ionide-vscode-fsharp#1042]
I am using SqlDataProvider and it both builds and is well parsed by ionide on Windows. It builds from command line on Mac, but ionide cannot seem to parse it.

The error:

> The type 'Object' is required here and is unavailable. You must add a reference to assembly 'System.Private.CoreLib, Version=4.0.0.0, Culture=neutral, PublicKeyToken=7cec85d7bea7798e'.

![type provider os x corelib](https://user-images.githubusercontent.com/731365/54018566-b2f56200-4191-11e9-8952-db436effe82c.png)

Contrary to issues [1026](https://github.com/ionide/ionide-vscode-fsharp/issues/1026) and [992](https://github.com/ionide/ionide-vscode-fsharp/issues/992), I used msbuild instead of dotnet msbuild host and net fsac runtime instead of netcore, but with no difference.

After adding
>   <PropertyGroup>
    <FscToolPath>/Library/Frameworks/Mono.framework/Versions/Current/Commands</FscToolPath>
    <FscToolExe>fsharpc</FscToolExe>
  </PropertyGroup>
it even builds with `dotnet build` from a command line, but of course does not help with the ionide.

**Comment by dvitonis** [comment]
I contemplated this issue for a month now, but it took posting it here to find the solution.

I have set Fsac Runtime to `net` in **User Settings** rather than **Workspace Settings** (and pointed to the right mono) and now ionide resolves everything fine.

Since we are here, why does it make so much difference here the User vs Workspace settings? It may mean that I will have to be switching fsac when switching between the projects.
